model/project_parser.go

```diff
@@ -176,17 +176,27 @@ func (pds *parserDependencies) UnmarshalYAML(unmarshal func(interface{}) error)
 // UnmarshalYAML reads YAML into a parserDependency. A single selector string
 // will be also be accepted.
 func (pd *parserDependency) UnmarshalYAML(unmarshal func(interface{}) error) error {
-	if err := unmarshal(&pd.TaskSelector); err != nil {
-		return err
+	type copyType parserDependency
+	var copy copyType
+	if err := unmarshal(&copy); err != nil {
+		// try unmarshalling for a single-string selector instead
+		if err := unmarshal(&pd.TaskSelector); err != nil {
+			return err
+		}
+		otherFields := struct {
+			Status        string `yaml:"status"`
+			PatchOptional bool   `yaml:"patch_optional"`
+		}{}
+		// ignore error here: expected to fail considering the single-string selector
+		grip.Debug(unmarshal(&otherFields))
```

Comment:
I think that we should add context to this error or omit logging entirely. This log message will be hard to track.

Reply:
We can just ignore them. They are unlikely to be important but I could see the debugging being useful when debugging validation locally, and I saw it in a different function so. I'll just drop it though.